modules/router-certificates.adoc

```diff
@@ -0,0 +1,87 @@
+from the service-ca controller for its own metrics, and the service-ca
+controller puts the certificate in a secret named metrics-tls in the
+openshift-ingress-operator namespace. Additionally, the Ingress Operator
+requests a certificate for each Ingress Controller, and the service-ca
+controller puts the certificate in a secret named `router-metrics-certs-<name>`,
+where `<name>` is the name of the Ingress Controller, in the
+`openshift-ingress`. namespace.
+
+Each Ingress Controller has a default certificate that the Controller uses for
+secured routes that do not specify their own certificates. You can set a custom
+default certificate, or else the Operator generates a certificate for you. The
+Operator uses its own self-signed signing certificate to sign any default
+certificate that it generates. The Operator generates this signing certificate
+and puts it in a secret named router-ca in the openshift-ingress-operator
+namespace. When the Operator generates a default certificate, it puts the
+default certificate in a secret named router-certs-<name> (where <name> is the
+name of the Ingress Controller) in the openshift-ingress namespace.
+
+[WARNING]
+====
+The Operator generates a default certificate for an Ingress Controller to serve
+as a placeholder till you configure a custom default certificate. Do not use
+operator-generated default certificates in production clusters.
+====
+
+== Expiration
+
+The expiration terms for the Ingress Operator's certificates are as follow:
+
+* The expiration date for metrics certificates that the `service-ca` controller
+creates is two years after the date of creation.
+* The expiration date for the Operator's signing certificate is two years after
+the date of creation.
+* The expiration date for default certificates that the Operator generates is two
+years after the date of creation.
+
+You cannot specify custom expiration terms on certificates that the Ingress
+Operator or `service-ca` controller creates.
+
+You cannot specify expiration terms when installing {product-title} for
+certificates that the Ingress Operator or `service-ca` controller create.
+
+== Services that use it
+
+Prometheus uses the certificates that secure metrics.
+
+The Ingress Operator uses its signing certificate to sign default certificates
+that it generates for Ingress Controllers for which you do not set custom
+default certificates.
+
+Cluster components that use secured routes may use the default Ingress
+Controller's default certificate.
+
+Ingress to the cluster via a secured route uses the default certificate of the
+Ingress Controller via which the route is accessed unless the route specifies
+its own certificate.
+
+== How to update or extend it
+
+To force regeneration of a certificate, delete the corresponding secret.
+
+== Renewal
+
+The `service-ca` controller automatically rotates the certificates that it issues.
+However, it is possible to manually rotate service serving certificates.
```

Comment:
If we are going to say this we need to show a user how.

Reply:
In https://github.com/openshift/openshift-docs/pull/18254#discussion_r356893774, I suggested a hyperlink:

> The `service-ca` controller automatically rotates the certificates that it issues. However, it is possible to [manually rotate service serving certificates](../../blob/master/authentication/certificates/service-serving-certificate.adoc#manually-rotate-service-ca_%7Bcontext%7D).

Would that be adequate?

Reply:
How? What command does a user run, what action do they take?

Reply:
Oh, I see the link is broken now (it may have been wrong before).  I'm referring to the steps at https://docs.openshift.com/container-platform/4.2/authentication/certificates/service-serving-certificate.html#rotate-service-serving_service-serving-certificate, namely `oc delete secret <secret>`.  Is a hyperlink sufficient, or do we need to inline (or reproduce a variant of) the "Manually rotate the generated service certificate" module?

Reply:
@Miciah Thanks for all of your help! I think by calling out that command and linking to that section should cover it.